lnwire/query_channel_range.go

```diff
@@ -75,3 +75,9 @@ func (q *QueryChannelRange) MaxPayloadLength(uint32) uint32 {
 	// 32 + 4 + 4
 	return 40
 }
+
+// LastBlockHeight returns the last block height covered by the range of a
+// QueryChannelRange message.
```

Comment:
nit: note that the range is inclusive

Reply:
What about instances where `NumBlocks` is `math.MaxUint32`?